libethash-cl/CLMiner.cpp

```diff
@@ -658,6 +665,7 @@ bool CLMiner::initDevice()
     s << " Memory : " << dev::getFormattedMemory((double)m_deviceDescriptor.totalMemory);
     cllog << s.str();
 
+    cllog << "Memory in bytes : " << m_deviceDescriptor.totalMemory;
```

Comment:
Why this double output of memory info ?

Reply:
This is ok to me provided you combine this with the previous log, e.g. `" Memory: 1.12 GB (1120000001 B)"`.

Reply:
fixed in https://github.com/ethereum-mining/ethminer/pull/1824/commits/eb95ad3ff0c00561f54b881424141c031a677da2